### ISSUE TITLE
NTR: Uvular swelling (Face & neck)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7826]
New term request Uvular swelling (nadea3_190423200534): Uvula swollen  UMLS: 2729501

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000177
- label: Uvular swelling
- parents: [Abnormal uvula morphology (HP:0000172)](https://hpo.jax.org/app/browse/term/HP:0000172)
- pmid: 
- def: Abnormal enlargement of the uvula related to an abnormal accumulation of fluid.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7826
- synonyms:
